benchmarking/benchmark_config.py

```diff
@@ -144,7 +144,8 @@ def get_num_prompts(input_len, output_len, max_concurrency):
                             num_prompts=get_num_prompts(isl, osl, 1),
                         )
                         for isl, osl in BATCH_1_BENCHMARK_COMMON_ISL_OSL_PAIRS
-                        if (isl, osl, 1) not in perf_ref_task_runs.get(_device, [])
+                        if (isl, osl, 1) not in perf_ref_task_runs.get(_device, []) 
+                        and isl <= model_spec.device_model_spec.max_context
```

Comment:
```suggestion
                        and (isl + osl) <= model_spec.device_model_spec.max_context
```
It should be the total `isl + osl` value that is compared to the max_context